docs/js-core-1/week-2/homework.md

```diff
@@ -114,3 +114,19 @@ Have some extra time before our next class? Fill it with these harder challenges
 **Alert:** Some of these challenges include concepts that we haven't taught you yet. This is a common experience as a professional developer, so start learning how to handle it today. Use Google, talk to your network, share your blockers, and work the problem.
 
 https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2
+
+## 9) Resolving conflicts (PD - 7 min)
+
+Watch [this 7-minute video](https://www.youtube.com/watch?v=qDfSYz0PX9g) on Resolving Workplace Conflicts.
+
+## 10) Writing an email for an internship (PD - 1 hr)
+
+Draft an email to apply for an internship at a fictional company called Vonda. Make sure to include the five things below:
+
+1. Why you want to become a software developer
+2. Your background and your past
```

Comment:
Can we make a note around this? For some of our trainees, explaining their background and their past is a huge request. It's quite a casual demand here. Could we say something like

```suggestion
2. Your background and your past (as much as you are comfortable sharing)
```